metamod/game_support.cpp

```diff
@@ -146,12 +148,10 @@ mBOOL DLLINTERNAL setup_gamedll(gamedll_t *gamedll) {
 	static char autodetect_desc_buf[NAME_MAX]; // pointer is given outside function
 	char install_path[NAME_MAX];
 	const game_modinfo_t *known;
-#ifdef _WIN32
-	const char* cp;
-#elif defined(__linux__)
-	const char* cp, * strippedfn;
+#ifdef __linux__
+        char *strippedfn;
```

Comment:
Because [`loc` is `char *`](https://github.com/Bots-United/metamod-p/blob/0c33cf5ac498aaa44fa75bcc0d816614ec6ea0c3/metamod/game_support.cpp#L201). And it requires too many chages to fix it.
___

I already have patch where `strippedfn` is `const char *`. But now it required to rebase. One of reasons: i need to minimize space changings. I am not being sarcastic. =)
I will send it later.